packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -34,6 +33,9 @@ import {
 import { startVaultFactory } from '../../src/econ-behaviors.js';
 import '../../src/vaultFactory/types.js';
 import * as Collect from '../../src/collect.js';
+import { calculateCurrentDebt } from '../../src/interest-math.js';
+
+// #region Support
```

Comment:
I think you're doing the same thing I do this way.  (You can see more instances of `//////` in this file.
```suggestion
// Support /////////////////////////////////
```

Reply:
Basically, though `#region` is understood by VS Code to fold: https://code.visualstudio.com/docs/editor/codebasics#_folding

Looks like WebStorm can be configured to do this too: https://www.jetbrains.com/help/webstorm/working-with-source-code.html#use-the-surround-with-action